extras/create_send_notifications.py

```diff
@@ -43,7 +43,8 @@ def create_output_directories():
     )
 
 def build_notifications_org_list(db):
-    """Return notifications organization list.
+    """Return list of notifications to generate and 
+       notifications not generated.
```

Comment:
I think this is clearer and more correct since the lists contain org IDs, not notifications:
```suggestion
    """Return a list of org IDs for which notifications should be
    generated as well as a list of org IDs for which notifications
    should not be generated.
```

It's a bit wordy, though, so maybe someone else can think of something more succinct.